README.md

```diff
@@ -76,7 +76,7 @@ Usage
 ### Updating sub-levels `composer.json` files
 
 
-In order for Composer Merge Plugin to install dependencies from updated or newly created sub-level `composer.json` files in your project you need to run the command:
+for Composer Merge Plugin to install dependencies from updated or newly created sub-level `composer.json` files in your project you need to run the command:
```

Comment:
Accidental removal?

Reply:
Accidental Removals butthis could be a professional things .
and i found this Mistake so  please merge this.